Xamarin.Forms.Controls/TestCases.cs

```diff
@@ -192,9 +193,15 @@ public void FilterTracker(IssueTracker tracker)
 				FilterIssues(_filter);
 			}
 
-			public void FilterIssues(string filter = null)
+			public async void FilterIssues(string filter = null)
 			{
+				filter = filter?.Trim();
 				_filter = filter;
+				
+				// Deeeee bounce
+				await Task.Delay(10);
```

Comment:
don't filter as the search is typed. This reduces the amount of list changes and speeds up automation